app/src/main/java/io/github/rosariopfernandes/notdog/MainActivity.kt

```diff
@@ -42,51 +46,113 @@ class MainActivity : AppCompatActivity() {
     companion object {
         const val CAMERA_REQUEST = 2018
         const val CAMERA_PERMISSION = 100
+        const val GALLERY_REQUEST = 2019
+        const val GALLERY_PERMISSION = 101
+        const val IMAGE_PICKER_INTENT_TYPE = "image/*"
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         hideSystemUI()
 
-        fab.setOnClickListener {
+        setCameraOnClickListener()
+        setGalleryOnClickListener()
+    }
 
+    private fun setCameraOnClickListener() {
+        fabCamera.setOnClickListener {
             permissinaManager {
                 activity = this@MainActivity
                 permission = Manifest.permission.CAMERA
                 permissionExplaination = getString(R.string.permission_explanation)
                 requestCode = CAMERA_PERMISSION
                 then {
-                    val cameraIntent = Intent(android.provider.MediaStore.ACTION_IMAGE_CAPTURE)
+                    val cameraIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
                     startActivityForResult(cameraIntent, CAMERA_REQUEST)
                 }
             }
+        }
+    }
 
+    private fun setGalleryOnClickListener() {
+        fabGallery.setOnClickListener {
+            permissinaManager {
```

Comment:
I think we could get rid of this library. Maybe not on a different PR, but I've opened an issue to track it: #61

Reply:
Yeah sounds good to me

Reply:
Can you assign that issue to me I can pick that up

Reply:
It's not letting me assign it to you, can you please comment on it maybe? I think it's a bug with GitHub.